tests/ludwig/features/test_text_feature.py

```diff
@@ -124,5 +134,9 @@ def test_get_decoded_targets_and_predictions():
         decoded_predictions,
     ) = text_feature.get_decoded_targets_and_predictions(targets, predictions, tokenizer)
 
-    assert decoded_targets == [" first she 18 yearman our<s>"]
-    assert decoded_predictions == [" first she 18 yearman our<s>"]
+    assert isinstance(decoded_targets, list)
+    assert isinstance(decoded_predictions, list)
+    assert all(isinstance(x, str) for x in decoded_targets)
+    assert all(isinstance(x, str) for x in decoded_predictions)
+    assert decoded_targets == decoded_predictions
```

Comment:
@amankhandelia One small question: would it work if we said `assert decoded_targets == decoded_predictions == decoded_texts` for concision?  Thank you.

Reply:
yes, that would work, I have made the change to that effect

Reply:
@amankhandelia Thank you very much for making the change.  At this point, I want to consult with a teammate (just to be 100% sure) -- so I will get back to you within 12 -- 13 hours from now.  Thanks for your patience!